migrations/20220808163441_262_authn_providers__saml__add.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO authn_providers (id, name) VALUES (5, 'SAML');
```

Comment:
TODO: make this idempotent?

Reply:
@mwest1066 should this work?

```suggestion
INSERT INTO authn_providers (id, name) VALUES (5, 'SAML') ON CONFLICT DO NOTHING;
```

Reply:
Yep, should work.